src/components/LoginNav/index.js

```diff
@@ -118,17 +118,31 @@ const LoginNav = ({
   return (
     <div className={styles.loginContainer}>
       {loggedIn ? renderLoginPanel() : (
-        <a
-          href='javascript:void(0)'
-          onClick={(event) => {
-            const retUrl = encodeURIComponent(window.location.href)
-            window.location = authURLs.location.replace('%S', retUrl).replace('member?', '#!/member?')
-            event.preventDefault()
-            return false
-          }}
-        >
+        <React.Fragment>
+          <a
+            className={styles.loginBtn}
+            href='javascript:void(0)'
+            onClick={(event) => {
+              const retUrl = encodeURIComponent(window.location.href)
+              window.location = authURLs.location.replace('%S', retUrl).replace('member?', '#!/member?')
```

Comment:
Please change URL in **community-app** config file instead use `replace`:
```
replace('member?', '#!/member?')
```

https://github.com/topcoder-platform/community-app/blob/d56f801a9ff5341b5499fb53637c52951b28cff4/config/default.js#L380

Reply:
@luizrrodrigues you mean here, we don't do the replace call?

Reply:
@skyhit Yes is change the URL in config file don't need to run second replace call:
```
replace('member?', '#!/member?')
```

Reply:
@luizrrodrigues fixed, please check again, we'd like to merge as quickly as we can

Reply:
Make sure to re-run `npm run build` to include updated `dist` folder

Reply:
@skyhit ☝️

Reply:
@luizrrodrigues done